src/libs/ReportUtils.ts

```diff
@@ -1799,8 +1808,11 @@ function getTransactionDetails(transaction: OnyxEntry<Transaction>, createdDateF
  * - in case of expense report
  *    - the current user is the requestor and is not settled yet
  *    - or the user is an admin on the policy the expense report is tied to
```

Comment:
specify approver of the policy as well along with admin here? like

or the user is an approver or admin on the policy the expense report is tied to